api/litmuschaos/v1alpha1/chaosengine_types.go

```diff
@@ -119,6 +119,33 @@ type ApplicationParams struct {
 	AppKind string `json:"appkind,omitempty"`
 }
 
+type Selector struct {
+	Workloads []Workload `json:"workloads,omitempty"`
+	Pods      []Pod      `json:"pods,omitempty"`
+}
+
+type WorkloadKind string
+
+const (
+	WorkloadDeployment       WorkloadKind = "deployment"
+	WorkloadStatefulSet      WorkloadKind = "statefulset"
+	WorkloadDaemonSet        WorkloadKind = "daemonSet"
+	WorkloadDeploymentConfig WorkloadKind = "deploymentconfig"
```

Comment:
Should we use the camel case for `deploymentconfig` value?

Reply:
two reasons to add this in lowercase:
1. to maintain it uniform across repos as we are using lower-case letters for resources in other repos
2. we do string equality comparisons by converting them into lowercase so maintaining them in lower case bydefault